legion/legion/edi/server.py

```diff
@@ -318,7 +319,8 @@ def generate_token():
         jwt_life_length = timedelta(minutes=int(app.config['JWT_CONFIG']['jwt.length.minutes']))
```

Comment:
@aliaksandr-d  is token TTL configured globally for all tokens? It should be defined individually of each token